torch/onnx/_globals.py

```diff
@@ -31,6 +32,14 @@ def __init__(self):
         self.operator_export_type: Optional[_C_onnx.OperatorExportTypes] = None
         self.onnx_shape_inference: bool = True
 
+        # Internal feature flags
+        if os.getenv("TORCH_ONNX_EXPERIMENTAL_RUNTIME_TYPE_CHECK") == "1":
```

Comment:
nit: perhaps the environmental variable values can be set to the same as enum strings ("DISABLED", etc)?

Reply:
Done